blog/data/authors/norbert.kaminski.json

```diff
@@ -1,5 +1,5 @@
 {
     "name": "Norbert Kamiński",
-    "bio": "Junior Embedded Systems Engineer at 3mdeb. Always thirst for knowledge, now focused on Linux embedded systems. His interests include Rust language, microcontroller programming and hardware design.",
+    "bio": "Junior Marketing Specialist at 3mdeb. With a passion for continuous learning, I'm always eager to acquire new skills and expand my knowledge in the field of marketing.",
```

Comment:
@Asiderr also change the info in the .md file according to this guide: https://github.com/3mdeb/news-and-ideas#add-new-profile-page

Reply:
I added required changes. 083a71e